### ISSUE TITLE
Delete old log-files

### ISSUE THREAD

**Issue by Satlinker** [linuxmint/timeshift#311]
Hi, there are tons of old log files with btrfs backups. Is there a need to hold logs older than the oldest backup?

**Comment by eddiem5** [comment]
Love timeshift but it does generate a bunch of log files and I was wondering how to manage them also...

**Comment by Satlinker** [comment]
You can delete them manually in the menu: show time shift protocols, then dolphin opens as root. But it makes no sense to keep the log files longer than the oldest backup.

**Comment by eddiem5** [comment]
Of course, I can delete manually but I like set and forget it kind of stuff - much like time shift..

Why not just add the logging directory to log rotate??  Any reason why not??
I am going to look at that now....

**Comment by ygerlach** [comment]
i guess the logs of creating a snapshot should be stored and deleted with the specific snapshot, in case there is something wrong or unexpected with the snapshot it can be diagnosed.

Other than that, i think this might be a thing for a systemd-tempfile.d entry, to clean files older than some value. Or let timeshift itself clean them.
